students/idcrickmore/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,16 @@
+def fizzbuzz(x):
```

Comment:
python modules (the filename) should be all lower case separated by underscores (snake case) -> `fizz_buzz.py`